bin/_mocha

```diff
@@ -206,19 +206,6 @@ if (program.reporterOptions !== undefined) {
 
 mocha.reporter(program.reporter, reporterOptions);
 
-// load reporter
-
-var Reporter = null;
-try {
-  Reporter = require('../lib/reporters/' + program.reporter);
-} catch (err) {
-  try {
-    Reporter = require(program.reporter);
-  } catch (err) {
-    throw new Error('reporter "' + program.reporter + '" does not exist');
-  }
-}
-
```

Comment:
This all appears to be unnecessary since 191b88c079d44f527dfeb022fbdb4079440c8428